sdk/subscription/Microsoft.Azure.Management.Subscription/tests/NetCore/InMemoryTests/SubscriptionTests.InMemory.cs

```diff
@@ -23,8 +26,157 @@ public SubscriptionClient GetSubscriptionClient(RecordedDelegatingHandler handle
             HttpMockServer.Mode = HttpRecorderMode.Playback;
             return client;
         }
+        
+        /*[Fact]
```

Comment:
Can we delete the commented test?

Reply:
Uncommented and executed the test. It works!